docs/cli/cli.md

```diff
@@ -28,10 +28,12 @@ The AWS Amplify CLI uses AWS CloudFormation and nested stacks. This allows you t
 
 For example, running `amplify add auth` will bootstrap a CloudFormation template in the `amplify>auth` folder. The infrastructure template can be re-used in CI/CD processes (either with the Amplify Console or your own build process), and can be used to replicate stacks.
 
+**Warning:** We strongly advise you not to manually update or delete resources created by the AWS CloudFormation, it may cause the stack to stuck in a state that can no longer be updated. It can be extremely hard to recover even with the help of AWS internal engineering teams. Among those operations, manually updating or deleting **Cognito** resources (they are created by the Amplify Auth category) are considered especially dangerous, it is even harder to recover, and you may loose user data or break auth related functionalities in your app. 
```

Comment:
1. "Note:" Instead of Warning to keep it consistent with the rest of the docs.
2. Wordsmithed: We strongly advise against manually updating or deleting resources created by the AWS CloudFormation; it may cause the stack to be stuck in a state that can no longer be updated.
3. Delete this sentence: It can be extremely hard to recover even with the help of AWS internal engineering teams. 
4. Updated: Manually updating or deleting **Cognito** resources (created by the Amplify Auth category) are considered especially dangerous and you may lose user data or break auth-related functionalities in your app.